src/style.css

```diff
@@ -0,0 +1,54 @@
+h1,
+h2 {
+  color: orange;
+  text-align: center;
+}
+
+body {
+  font-family: sans-serif;
+}
+
+#input {
+  padding: 10px;
+  border: 1px solid #ddd;
+  border-radius: 6px;
+}
+
+#button {
+  padding: 12px;
+  background-color: #5058ab;
+  color: white;
+  border: none;
+  border-radius: 6px;
+  cursor: pointer;
+}
+
+ul {
+  list-style: none;
+  padding: 0;
+}
+
+li {
+  display: flex;
+  align-items: center;
+  padding: 8px;
+  border-bottom: 1px solid #dbd9d9;
+}
+
+li:last-child {
+  border-bottom: none;
+}
+
+li button {
+  margin-left: auto;
+  padding: 4px;
+  background-color: #bba4a4;
+  color: white;
+  border: none;
+  border-radius: 45%;
+  cursor: pointer;
+}
+
+li button:hover {
+  background-color: #c47c6dec;
+}
```

Comment:
- [ ] Please style your project to match the [captured video](https://www.youtube.com/watch?v=AcUd-_Yjjqg) design structure.
**Given Design**
![fix](https://user-images.githubusercontent.com/55596970/231935667-1efd01d8-e99e-4605-8d7f-faba7c2fa1cc.png)
**Your Design**|

![fix2](https://user-images.githubusercontent.com/55596970/231935980-9fe8c7f2-7d40-481b-980c-6144172ad83f.png)